docs/hydration.md

```diff
@@ -165,3 +165,14 @@ will be hydrated *inside* the corresponding values for that key.
   [:a [:b :c] :e])
 ;; -> {:a {:b {:c 1} :e 2}}
 ```
+
+## Refreshing Hydrations
+
+Simple and batch hydration functions are lazily resolved within loaded namespaces on first call of `hydrate`. If hydrate
+has been called, which results in resolution of new models which contain hydrations in the same namespace; these new
+hydrations will not be evaluated. Invoking `refresh!` will clear the previously resolved hydration functions (resulting
+in a need to re-scan the namespaces on next hydrate).
+
+```clojure
+(refresh!)
+```
```

Comment:
Allows users to clear the hydration functions (forcing a re-scan on next `hydrate`).

This is useful for when models are resolved over time post-hydrate

Not sure how we can wire-it in to be automatic. I would love to put it where the model resolves, but it would end up as a cyclic dependency. Thoughts?